lib/data/monorepo.json

```diff
@@ -642,6 +642,7 @@
       "/^com.oracle.database.jdbc:/",
       "/^com.oracle.database.nls:/"
     ],
+    "oxlint-eslint": ["oxlint", "eslint-plugin-oxlint"],
```

Comment:
Maybe "oxlint" is better for this? If someone uses _only_ oxlint, they will get a strange "Update exlint-eslint to v1.1.0" PR due to this grouping rule.